### DISCUSSION TITLE
Dubbi giustificazione assenze ed eventi vari

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#369] [Idee e proposte]
Buongiorno,i genitori stanno riscontrando diverse perplessità ,in quanto giustificano l'assenza , ma ancora una volta gli appare la scritta modifica e ci stanno tartassando di chiamate e noi dobbiamo spiegare che esce quella scritta fino a quando il docente non convalida l'assenza /ritardo.A quanto pare stiamo ricevendo quasi tutti i giorni chiamate con le stesse richieste.A questo punto rinnovo la richiesta di giustificare una volta per tutte e basta come in tutti i registri  senza aspettare la convalida da parte del docente perché non è possibile andare avanti così.

**Comment by trinko** [comment]
Dal momento che nella giustificazione il genitore può scrivere un testo libero, mi pare naturale che il docente vada a controllare la motivazione. Anche perché una motivazione potrebbe non essere considerata una giustificazione valida.
La validazione serve semplicemente a questo: a segnare cosa è stato controllato e cosa no. Se i docenti vanno a validare le giustificazioni ogni giorno, il problema indicato non si pone.
